developer/src/kmc-kmn/test/kmw/test-kmw-messages.ts

```diff
@@ -0,0 +1,74 @@
+import 'mocha';
+import { assert } from 'chai';
+import { KmwCompilerMessages } from '../../src/kmw-compiler/kmw-compiler-messages.js';
+import { TestCompilerCallbacks, verifyCompilerMessagesObject } from '@keymanapp/developer-test-helpers';
+import { makePathToFixture } from '../helpers/index.js';
+import { KmnCompiler } from '../../src/main.js';
+import { CompilerErrorNamespace } from '@keymanapp/common-types';
+
+describe('KmwCompilerMessages', function () {
+  const callbacks = new TestCompilerCallbacks();
+
+  it('should have a valid KmwCompilerMessages object', function() {
+    return verifyCompilerMessagesObject(KmwCompilerMessages, CompilerErrorNamespace.KmwCompiler);
```

Comment:
Maybe busy work, but what about splitting `validate_gesture.keyman-touch-layout` into two separate touch keyboards:
* flick
* multi-tap

to verify the warning message is generated for each?

Reply:
Yeah felt like busywork and I figured it was enough for now. It would be low priority.